crates/subspace-node/src/commands/run/domain.rs

```diff
@@ -468,6 +477,13 @@ pub(super) async fn run_domain(
         .chain_constants(consensus_best_hash)
         .map_err(|err| Error::Other(err.to_string()))?;
 
+    let consensus_network_sync_oracle_wrapper = Arc::new(ConsensusChainSyncOracleWrapper::new(
+        consensus_network_sync_oracle,
+        consensus_chain_sync_params
+            .as_ref()
+            .map(|params| params.snap_sync_orchestrator.domain_snap_sync_finished()),
+    ));
```

Comment:
Still don't really understand why consensus syncing oracle needs to check if domain snap sync hash finished...